test/basic.tests.js

```diff
@@ -523,4 +524,172 @@ contract('LiquidityBridgeContract', async accounts => {
         expect(lbcBal).to.be.a.bignumber.eq(initialLPCol);
         expect(web3.utils.toBN(0)).to.be.a.bignumber.eq(finalLPCol);
     });
+
+    it('should register pegout', async () => {
+        const callData = web3.eth.abi.encodeFunctionCall(mock.abi[0], ['99'])
+        const getBalances = () => Promise.all([
+            instance.getBalance("0xf2af87AEB573cdD4B83d543588F85C5280d1cCB5"),
+            instance.getPegOutBalance("0xf2af87AEB573cdD4B83d543588F85C5280d1cCB5"),
+            web3.eth.getBalance(instance.address)
+        ]);
+
+        const [
+            userPegInBalanceBefore,
+            userPegOutBalanceBefore,
+            contractBalanceBefore
+        ] = await getBalances();
+
+        let quote = utils.getTestQuote(
+            instance.address, //lbc address
+            "0xf2af87AEB573cdD4B83d543588F85C5280d1cCB5",
+            callData,
+            liquidityProviderRskAddress,
+            "0xf2af87AEB573cdD4B83d543588F85C5280d1cCB5",
+            web3.utils.toBN(1)
+        );
+        const msgValue = quote.val.add(quote.callFee);
+
+        const quoteHash = await instance.hashQuote(utils.asArray(quote));
+        const signature = await web3.eth.sign(quoteHash, liquidityProviderRskAddress);
+        const pegOut = await instance.registerPegOut(utils.asArray(quote), signature, {value: msgValue});
+        truffleAssertions.eventEmitted(pegOut, "PegOut");
+
+        const [
+            userPegInBalanceAfter,
+            userPegOutBalanceAfter,
+            contractBalanceAfter
+        ] = await getBalances();
+
+        expect(userPegInBalanceBefore.toString()).to.be.eq(userPegInBalanceAfter.toString());
+        expect(userPegOutBalanceBefore.toString()).to.not.be.eq(userPegOutBalanceAfter.toString());
+        expect(contractBalanceBefore).to.not.be.eq(contractBalanceAfter);
```

Comment:
contractBalanceAfter should be equal to contractBalanceBefore + the value sent by the user.

Reply:
done